packages/types/src/runtime/staking/types.spec.ts

```diff
@@ -15,5 +15,17 @@ describe('RewardDestination', (): void => {
     expect(destinationController.isController).toBeTruthy();
     const destinationControllerStr = createType(registry, 'RewardDestination', 'controller');
     expect(destinationControllerStr.isController).toBeTruthy();
+
+    const destinationAccountAddress = '5FEe8Ht1ZTzNjQcvrxbLxnykA2EXfqN5LMog2gaNPus4tfZR';
+    const destinationAccount = createType(registry, 'RewardDestination', {account: destinationAccountAddress}, 2);
+    expect(destinationAccount.asAccount.toString()).toEqual(destinationAccountAddress);
+    expect(destinationAccount.isAccount).toBeTruthy();
+    const destinationAccountStr = createType(registry, 'RewardDestination', {account: destinationAccountAddress});
+    expect(destinationAccountStr.asAccount.toString()).toEqual(destinationAccountAddress);
+    expect(destinationAccountStr.isAccount).toBeTruthy();
+
+    const destinationPayee = new RewardDestination(registry, {account: destinationAccountAddress});
```

Comment:
nit:
```suggestion
    const destinationPayee = new RewardDestination(registry, { account: destinationAccountAddress });
```

Reply:
I though eslint would do this so not sure why it's different

Reply:
I will search which rule will do this and enable it explicitly in the eslintrc..

Reply:
"object-curly-spacing": [2, "always"] -- if I add this rule, I have 808 errors.. I will update this PR without the rule and modify at these two place and have another PR to fix those 808 errors.. there as option to fix it with code..will use that..